File: Tugas coding modul 4/coding py/PRAK402-2310817120005-Noviana_Nur_Aisyah.py
apasaja = int(input('Input \n'))

print('\nOutput')
for i in range(1, apasaja, +1):
    if i % 2 != 0:
        print(i, end=" ")

print('')
for i in range(apasaja, 0, -1):
    if i % 2 == 0:
        print(i, end=" ")

File: Tugas coding modul 4/coding c/PRAK402-2310817120005-Noviana_Nur_Aisyah.c
#include <stdio.h>

void main() {
    int in, i;
    printf("Input \n");
    scanf("%d", &in);

    printf("\nOutput \n");
    for (i = 1; i <= in; i++) {
        if (i % 2 != 0) {
            printf("%d ", i);
        }
    }

    printf("\n");
    for (i = in; i > 0; i--) {
        if (i % 2 == 0) {
            printf("%d ", i);
        }
    }
}